File: Beginner/age-check.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
  int age;
   cout<<"Enter the age"<<endl;
   cin>>age;
   if(age < 10 )
      {
        cout<<"child";
       }
   else if ( age > 10 && age <= 18 )
       {
         cout<<"teen" ;
       }
   else {
          cout<<"adult";

        }

     return 0;
}

File: Beginner/ageCheck.py
######################         Code By - ORASHAR          #########################
###################### Age check fro child, teen or adult #########################

try:
    n = int(input("Enter  the number: "))
    if n < 10:
        print("child")
    elif n <= 18:
        print("teenager")
    else:
        print("adult")

except:
    print("Invalid input!")
